packages/manager/modules/pci/src/projects/project/public-gateways/service/index.js

```diff
@@ -0,0 +1,9 @@
+import angular from 'angular';
+
+import service from './public-gateways.service';
+
+const moduleName = 'ovhManagerPciPublicGatewaysService';
+
+angular.module(moduleName, []).service('PciPublicGatewaysService', service);
```

Comment:
No dependencies ? No ui-kit, agular-translate etc ?

Reply:
I have created this module only to include the service. Since this service can be re-used in other products like Public-gateways and Additional IPs, to avoid including the entire module as a dependency in other products, we can only include the new service module. So no other dependencies are required in this module.